### PR TITLE
Initial dash-cytoscape R package build

### PR DESCRIPTION
**Pull request by rpkyle**
## About
This PR proposes to commit an initial build of the `dashCytoscape` R package.

## Description of changes 
There are some minor additions to the `package.json` metadata, as well as generated R package source and online documentation.

## Pre-Merge checklist
- [x] If there was any conflict, it was solved correctly
- [x] All changes were documented in CHANGELOG.md.
- [x] All tests on CircleCI have passed.
- [x] All Percy visual changes have been approved.
- [x] Two people have :dancer:'d the pull request. You can be one of these people if you are a Dash Cytoscape core contributor.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   the node's and edge's dictionaries, respectively (e.g., 'classes', 'positions', etc.).
 * Removed `Tree`'s method `add_child`, because it is redundant with `add_children` called with an
   argument of length 1.
+* Contributed initial build of R package.
```
Comment (shammamah) [inline CHANGELOG.md]: This could probably go in "Added"!

File: DESCRIPTION [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Package: dashCytoscape
+Title: A Component Library for Dash aimed at facilitating network visualization in Python, wrapped around Cytoscape.js
+Version: 0.1.1
+Authors @R: as.person(c(The Plotly Team <xinghan@plot.ly>))
```
Comment (shammamah) [inline DESCRIPTION]: Should we have a cytoscape@plot.ly email too?

Comment (rpkyle) [reply DESCRIPTION]: Good idea! Will set that up.

File: CHANGELOG.md (hunk 1 above)
Comment (rpkyle) [reply CHANGELOG.md]: Right, that makes more sense. Will migrate this bit.

Comment (rpkyle) [reply CHANGELOG.md]: fixed in c969f1e

File: DESCRIPTION (hunk 2 above)
Comment (rpkyle) [reply DESCRIPTION]: fixed in 1fa98d1

**Comment by rpkyle** [comment]
💃

**Pull request merged**